diff -ur kmobiletools-0.4.3.1.org/kmobiletools/kmobiletools/c_config.cpp kmobiletools-0.4.3.1/kmobiletools/kmobiletools/c_config.cpp
--- kmobiletools-0.4.3.1.org/kmobiletools/kmobiletools/c_config.cpp	2005-03-21 19:25:47.000000000 +0100
+++ kmobiletools-0.4.3.1/kmobiletools/kmobiletools/c_config.cpp	2006-01-07 14:33:50.265625250 +0100
@@ -85,10 +85,14 @@
 		return B921600;
 		case 8:
 		return B1500000;
+#ifdef B3000000
 		case 9:
 		return B3000000;
+#endif
+#ifdef B4000000
 		case 10:
 		return B4000000;
+#endif
 	}
 	return 0;
 }
