--- kmobiletools/kmobiletools/kmobiletools.desktop	2006-08-19 03:05:45.000000000 +0200
+++ kmobiletools/kmobiletools/kmobiletools.desktop-new	2006-08-19 18:47:17.803627024 +0200
@@ -33,3 +33,4 @@
 Comment[sk]=KDE Mobilné telefóny
 Comment[sv]=KDE:s mobiltelefonsvit
 Terminal=false
+Categories=Qt;KDE;Network;
--- kmobiletools/kbluetoothpairingwizard/kbluetoothpairingwizard.desktop	2006-08-19 03:05:45.000000000 +0200
+++ kmobiletools/kbluetoothpairingwizard/kbluetoothpairingwizard.desktop-new	2006-08-19 19:03:26.869306648 +0200
@@ -18,3 +18,4 @@
 Comment[nl]=Een eenvoudige KDE-toepassing
 Comment[pt]=Uma aplicação simples do KDE
 Comment[sv]=Ett enkelt KDE-program
+Categories=Qt;KDE;Network;
--- kmobiletools/kmtsetup/kmtsetup.desktop	2006-08-19 03:05:45.000000000 +0200
+++ kmobiletools/kmtsetup/kmtsetup.desktop-new	2006-08-19 19:04:35.454880056 +0200
@@ -14,3 +14,4 @@
 Comment[ja]=kmobiletools で使用するデバイスを設定するためのアプリケーション
 Comment[nl]=Een toepassing voor het instellen van apparaten, gebruikt door KMobileTools
 Comment[pt]=Uma aplicação de configuração dos dispositivos usados pelo KMobiletools
+Categories=Settings;
