--- configure.in.in.orig	2007-04-06 20:24:51.000000000 +0200
+++ configure.in.in	2007-04-07 00:42:52.379942231 +0200
@@ -65,18 +65,22 @@
 
 KDE_CHECK_HEADER(p2kmoto.h, have_p2k=yes, have_p2k=no)
 AM_CONDITIONAL(with_p2kmoto, test $have_p2k = yes)
-KDE_CHECK_HEADER(obexftp/client.h, have_obexftp=yes, have_obexftp=no)
-AM_CONDITIONAL(compile_obex, test $have_obexftp = yes)
 
 if test "x$have_p2k" = "xyes"; then
 	AC_DEFINE(HAVE_P2KLIB, 1, [p2k filesystem support])
 	AC_SUBST(HAVE_P2KLIB)
 fi
 
-if test "x$have_obexftp" = "xyes"; then
-	AC_DEFINE(HAVE_OBEXFTP, 1, [ObexFTP filesystem support])
-	AC_SUBST(HAVE_OBEXFTP)
+AC_ARG_WITH(obexftp, AC_HELP_STRING([--without-obexftp], [Build kmobiletools without filesystem integration (default: check)]),
+        [with_obexftp=$withval], [with_obexftp=yes])
+if test "$with_obexftp" = "yes"; then
+	KDE_CHECK_HEADER(obexftp/client.h, have_obexftp=yes, have_obexftp=no)
+	if test "$have_obexftp" = "yes"; then
+		AC_DEFINE(HAVE_OBEXFTP, 1, [ObexFTP filesystem support])
+		AC_SUBST(HAVE_OBEXFTP)
+	fi
 fi
+AM_CONDITIONAL(compile_obex, test $have_obexftp = yes)
 
 # AC_ARG_ENABLE(kioslaves, AC_HELP_STRING([--enable-kioslaves], [enable KIOSlaves for filesystem access (default: disable)]),
 #     [enable_kioslaves=$enableval], [enable_kioslaves=no])
